pkgs/applications/video/olive-editor/default.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, fetchFromGitHub, pkgconfig, which, qmake, mkDerivation,
-  qtbase, qtmultimedia, frei0r, opencolorio, ffmpeg-full,
-  CoreFoundation  }:
+{ stdenv, fetchFromGitHub, pkgconfig, which, qmake, mkDerivation, qtbase,
+  qtmultimedia, wrapQtAppsHook, frei0r, opencolorio, ffmpeg-full,
```

Comment:
Do you need both qtbase & qtmultimedia? They're not explicitly used.

Otherwise, diff LGTM. Building locally currently.

Reply:
Yep, tested it we don't need qtbase when we already have qtmultimedia